### ISSUE TITLE
Why does your dll show up with 28 flags? 

### ISSUE THREAD

**Issue by mutschistink** [danielkrupinski/Osiris#3992]
some flags dont usually mean nothing but this seems very sus

https://www.virustotal.com/gui/file/4b987a15690e45e492e69a1d5e1fb945aee4d1a9588ce9f89a2ada86c7142b2d?nocache=1

@danielkrupinski

**Comment by Xsintashi** [comment]
false positive

**Comment by mutschistink** [comment]
> false positive also code is public, check if you re not sure or just dont use

mhm idk about the fact that there are 28 flags 

almost everybody on github is here bc they want stuff that they cannot code & understand themselves
that means the argument "check if youre not sure" isn't valid cus me and many other people can't prove shit if its safe or not

**Comment by Broihon** [comment]
bruh

**Comment by gr1ndy** [comment]
cheat dlls will always be flagged as viruses since they inject into a process and read and write into the game's memory in order to gain unfair advantage.

**Comment by Xsintashi** [comment]
> > false positive also code is public, check if you re not sure or just dont use
> 
> mhm idk about the fact that there are 28 flags
> 
> almost everybody on github is here bc they want stuff that they cannot code & understand themselves that means the argument "check if youre not sure" isn't valid cus me and many other people can't prove shit if its safe or not

don't use then

**Comment by mutschistink** [comment]
> cheat dlls will always be flagged as viruses since they inject into a process and read and write into the game's memory in order to gain unfair advantage.

yeah thats completly clear but that much? or do just most of the anti virus companies not like csgo hacks lol

**Comment by mutschistink** [comment]
> > > false positive also code is public, check if you re not sure or just dont use
> > 
> > 
> > mhm idk about the fact that there are 28 flags
> > almost everybody on github is here bc they want stuff that they cannot code & understand themselves that means the argument "check if youre not sure" isn't valid cus me and many other people can't prove shit if its safe or not
> 
> don't use then. Nobody cares if u use it or no.

k then 

but what is Orisis and Extra

and what is the best bypass cus you wrote that you need one

**Comment by notgoodusename** [comment]
> but what is Orisis and Extra
> 
> and what is the best bypass cus you wrote that you need one
OsirisAndExtra is a repo that uses osiris and adds more features like ragebot, fakelag, antiaim, etc
The bypass is needed as to not get vac banned, as such you need to download CSGhost on unknowncheats, obviously there are more injectors that disable vac but this is a really popular and safe one

**Comment by RaphielHS** [comment]
OsirisAndExtra is basically Osiris with extra utils and modules and (my own comment) is unstable and not fully optimized

**Comment by mutschistink** [comment]
> CSGhost

does it work easyier than turning off the wifi and doing it with CE ?

**Comment by notgoodusename** [comment]
> bro CE is safe what u talking about?
> 

CE gives vac almost instantly

**Comment by Broihon** [comment]
> 

My brother in christ, CE gives roughly the same amount of "flags" on VirusTotal like the binaries of this repo.

**Comment by diekunstderfuge** [comment]
> cheat dlls will always be flagged as viruses since they inject into a process and read and write into the game's memory in order to gain unfair advantage.

What do you mean by UNFAIR Advantage? Everybody else is cheating too...

**Comment by ImsilverDev** [comment]
it reads and writes game memory, ofc it will be flagged as a "virus" go ahead take a look at the source, that's why its open source, use your brain for a moment buckaroo

**Comment by Xsintashi** [comment]
> it reads and writes game memory, ofc it will be flagged as a "virus" go ahead take a look at the source, that's why its open source, use your brain for a moment buckaroo

author's thinking is github is for ppl who want stuff that they cannot code. So argument "check if youre not sure" isn't valid cus me and many other people can't prove shit if its safe or not 🙃

**Comment by ImsilverDev** [comment]
> > it reads and writes game memory, ofc it will be flagged as a "virus" go ahead take a look at the source, that's why its open source, use your brain for a moment buckaroo
> 
> author's thinking is github is for ppl who want stuff that they cannot code. So argument "check if youre not sure" isn't valid cus me and many other people can't prove shit if its safe or not 🙃
> 
> So if ure programmer, find another place than github cuz its for ppl who cant code

![image](https://user-images.githubusercontent.com/44844515/227785584-faa92bee-8f59-4860-b25a-83aec2455416.png)
